src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int RANDNUMBER = 100;
```

Comment:
Please find another name for this constant. Maybe MAX_NUMBER or NUMBERS_LIMIT

Reply:
the second day in our high-rise building in Kyiv there is no electricity and the Internet, the problem